pkg/sql/catalog/descpb/structured.go

```diff
@@ -250,6 +250,11 @@ func (desc *TableDescriptor) MaterializedView() bool {
 	return desc.IsMaterializedView
 }
 
+// IsRefreshViewRequired implements the TableDescriptor interface.
+func (desc *TableDescriptor) IsRefreshViewRequired() bool {
+	return desc.IsMaterializedView && desc.RefreshViewRequired
+}
```

Comment:
This isn't obvious but can you implement this method on `wrapper` in `tabledesc` instead please? We're trying to keep the amount of business logic in `descpb` to a minimum.